lib/grpc/GrpcService.ts

```diff
@@ -500,7 +500,7 @@ class GrpcService {
    * See [[Service.subscribeAddedOrders]]
    */
   public subscribeAddedOrders: grpc.handleServerStreamingCall<xudrpc.SubscribeAddedOrdersRequest, xudrpc.Order> = (call) => {
-    this.service.subscribeAddedOrders((order: Order) => {
+    this.service.subscribeAddedOrders(call.request.toObject(), (order: Order) => {
```

Comment:
What do you mean by `argv`? It shouldn't be a command line argument. I don't think you need an event either, I think you can just pass the request as you're doing here, then read it from the `subscribeAddedOrders` method and query the order book for all existing orders if necessary. I'll do a more thorough review later this weekend.

Reply:
@sangaman I was thinking of adding an option to toggle off streaming existing orders.

Reply:
Yeah we do want that, but there won't be argv here. We can add a command line argument to the `streamorders` cli command (I think it would make sense to default it to not stream existing orders), and then set it on the grpc request (the same way we set request parameters for other cli calls).

In here, the request should already have the field that determines whether to stream up front. It will be passed to the `subscribeAddedOrders` method, you can see how it's handled for other calls that have fields for the request. You just need to define the field in the proto file first.